SpatialGDK/Source/SpatialGDKFunctionalTests/Private/SpatialFunctionalTest.cpp

```diff
@@ -681,16 +681,16 @@ ASpatialFunctionalTestFlowController* ASpatialFunctionalTest::GetFlowController(
 			return FlowController;
 		}
 	}
-	return nullptr;
+	verifyf(false, TEXT("We should always find a flow controller.")) return nullptr;
```

Comment:
```suggestion
	verifyf(false, TEXT("We should always find a flow controller."));
	return nullptr;
```
I prefer treating them like functions, even if they are macros

Reply:
oh whoops, missed the semicolon